### ISSUE TITLE
[BUG] SfImage decorative images have red outline

### ISSUE THREAD

**Issue by rain2o** [vuestorefront/storefront-ui#1714]
**Describe the bug**
When SfImage is used for decorative images, without an alt text, there is a red outline set on it. I imagine this was originally added when the component required an alt text as a form of easily identifying images with the missing alt text. But now that the component supports decorative images, which must **not** have alt text, this becomes a problem we have to adjust for all of our decorative images.

If you still would like to try and enforce the developers to think about accessibility while using this component, a possible option would be to add an optional prop like `decorative` or `isDecorative` with a default value of `false`. This way you could use that as a check, and enforce an alt text if that isn't set to `true`. This way, as a developer using this component, we would have to intentionally tell the component it's a decorative image, pro-actively telling it not to enforce alt text standards. 

I don't know if that's a little more than what you intend to do within this component, but it's an option if you are looking to enforce this standard while still allowing decorative images.

**How to reproduce**
1. Use `SfImage` without an `alt` text.
2. View the page with the component to see the red outline.

**Expected behavior**
The image should be displayed as normal, without a red outline, unless the developer using the component intentionally sets the outline.

**Actual behavior**
Decorative images have a red outline.

**Code examples**
```vue
<SfImage
  src="https://via.placeholder.com/300x300.jpg"
  :width="300"
  :height="300"
/>
```

┆Issue is synchronized with this [Jira Zadanie](https://vsf.atlassian.net/browse/SFUI-41) by [Unito](https://www.unito.io)

**Comment by Baroshem** [comment]
Hi @rain2o 

I checked the code. This change was added 4 months ago so it's quite a recent change. However I think it's quite a good idea to have this `isDecorative` prop that will decide if the red outline should be applied. But I think it should be enabled by default so that it won't break the current setup, and if you would like the image to not have red outline you set a prop value to false. That way, for most users it will not be a change, and it will be a fix for your problem :)

Maybe you would like to start a PR with that change?
